lib/ui/views/cv_landing_view.dart

```diff
@@ -95,15 +94,9 @@ class _CVLandingViewState extends State<CVLandingView> {
       child: ListView(
         children: <Widget>[
           Container(
-            padding: const EdgeInsets.symmetric(vertical: 32),
-            child: Text(
-              'CircuitVerse',
-              textAlign: TextAlign.center,
-              style: Theme.of(context).textTheme.headline3.copyWith(
-                    fontFamily: 'Grinched',
-                    color: AppTheme.primaryColor,
-                  ),
-            ),
+            padding: const EdgeInsets.symmetric(vertical: 32, horizontal: 10),
+            child: Image.asset('assets/images/landing/cv_full_logo.png',
+                width: 90),
```

Comment:
Better formatting can be done here.

Reply:
Sure

Reply:
The code is formatted already using `flutter format`

Reply:
Done!